js/DOMelements.js

```diff
@@ -0,0 +1,10 @@
+var toast_section = document.getElementById('toast_section'),
```

Comment:
DOMelements.js file does not make particular sense, since you won't be de declaring all the elements in one file and using them in another in real work. 
I'd recommend you to refer to https://getbootstrap.com/docs/3.4/javascript/ 
just to take a look, at how they use components.